src/site/_includes/partials/paged.njk

```diff
@@ -47,3 +47,5 @@
   {% endif %}
 
 </nav>
+<web-actions actions="subscribe"></web-actions>
+{% include 'partials/subscribe.njk' %}
```

Comment:
I think we wouldn't have the subscribe form at the bottom of posts now. Instead there would just be a button at the bottom of pages that would link to /newsletter, which is where the subscribe form would be embedded.